.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # CODEOWNERS info: https://help.github.com/en/articles/about-code-owners
 # Owners are automatically requested for review for PRs that changes code
 # that they own.
-* @manishrjain @martinmr @jarifibrahim
+* @akon-dey @nosql22
```

Comment:
I would suggest you to add your user and me to this as well.

Reply:
Done